internal/services/oracledatabase/client/client.go

```diff
@@ -0,0 +1,26 @@
+// Copyright © 2024, Oracle and/or its affiliates. All rights reserved
+
+package client
+
+import (
+	oracedatabase "github.com/hashicorp/go-azure-sdk/resource-manager/oracledatabase/2024-06-01"
+	"github.com/hashicorp/go-azure-sdk/sdk/client/resourcemanager"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/common"
+)
+
+type Client struct {
+	OracleDatabaseClient *oracedatabase.Client
+}
+
+func NewClient(o *common.ClientOptions) (*Client, error) {
+	o.DisableCorrelationRequestID = true
```

Comment:
how come we are disabling this? please add a comment detailing why

Reply:
Good point - I've added a comment explaining why we disabled this.